target/linux/ramips/dts/mt7620a_bolt_bl201.dts

```diff
@@ -98,6 +110,7 @@
 		compatible = "jedec,spi-nor";
 		reg = <0>;
 		spi-max-frequency = <10000000>;
+		// m25p,fast-read;
```

Comment:
This comment does nothing, just remove it

Reply:
resolved: comment removed